.rubocop.yml

```
@@ -333,3 +336,158 @@ Performance/Sum:
 
 Style/StringChars:
   Enabled: true
+
+Gemspec/DevelopmentDependencies: # new in 1.44
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)